test cases/unit/25 shared_mod linking/meson.build

```diff
@@ -0,0 +1,6 @@
+project('shared library linking test', 'c', 'cpp')
+
+lib = shared_module('mylib',
+ 'libfile.c' # Split to different lines before and after the comma to test parser.
+ , install : false) # Don't install libraries in common tests; the path is platform-specific
```

Comment:
These comments and `install : false` are not needed for this test, so you can just remove them altogether to simplify the test.